.gitlab-ci.yml

```diff
@@ -742,6 +747,28 @@ subdir-build:
     - cuda
     - gpu
 
+# Ensure Ginkgo can be used when exporting the build directory
+export-build:
+  <<: *default_build
+  stage: code_quality
+  image: localhost:5000/gko-cuda102-gnu8-llvm8-intel2019
+  variables:
+    <<: *default_variables
+    BUILD_OMP: "ON"
+    BUILD_CUDA: "ON"
+    BUILD_HIP: "ON"
```

Comment:
Do we need both the HIP and CUDA modules to be compiled ? Can we not get away without them ?

Reply:
That's a good point, I think we can remove it from here and also from the subdir build.